website/docs/docs/integrate/dbt-semantic-layer.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "dbt Semantic Layer"
+id: dbt-semantic-layer
+description: "Introducing the dbt Semantic Layer"
+sidebar_label: "dbt Semantic Layer"
+---
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
```

Comment:
It might have more impact to choose just one of the sentences to end with an exclamation point.  Whichever point you think is more important to stress.